src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -2,36 +2,43 @@
 
 import static java.util.Objects.requireNonNull;
 import static seedu.address.logic.parser.CliSyntax.PREFIX_ID;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_MODULE_CODE;
 import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
 
 import seedu.address.commons.core.Messages;
 import seedu.address.model.Model;
+import seedu.address.model.person.ModuleCodeContainsKeywordsPredicate;
 import seedu.address.model.person.NameContainsKeywordsPredicate;
 import seedu.address.model.person.StudentIdContainsKeywordsPredicate;
 
 /**
- * Finds and lists all persons in address book whose name contains any of the argument keywords.
+ * Finds and lists all persons in address book whose name, student id or module code
+ * contains any of the argument keywords.
  * Keyword matching is case insensitive.
  */
 public class FindCommand extends Command {
 
     public static final String COMMAND_WORD = "find";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names contain "
-            + "the specified search term or matriculation number and displays them as a list with index numbers.\n"
-            + "Parameters: " + PREFIX_NAME + "STUDENT_NAME " + "or " + PREFIX_ID + "STUDENT_ID \n"
+            + "the specified search term, matriculation number or module code "
+            + "and displays them as a list with index numbers.\n"
+            + "Parameters: " + PREFIX_NAME + "STUDENT_NAME " + "or " + PREFIX_ID + "STUDENT_ID " + "or "
+            + PREFIX_MODULE_CODE + "MODULE_CODE\n"
             + "Example: " + COMMAND_WORD + " " + PREFIX_NAME + "John " + "or " + COMMAND_WORD + " "
-            + PREFIX_ID + "A0123456Z \n";
+            + PREFIX_ID + "A0123456Z " + "or " + COMMAND_WORD + " " + PREFIX_NAME + "CS2103T\n";
```

Comment:
```suggestion
            + PREFIX_ID + "A0123456Z " + "or " + COMMAND_WORD + " " + PREFIX_MODULE_CODE + "CS2103T\n";
```

I think this is a minor typo.